changelog.d/pr-3960.misc

```diff
@@ -0,0 +1 @@
+Issue templates: merge expected and actual results
```

Comment:
File name should not contain `pr-` prefix. It breaks the generated changelog:

```
 - Issue templates: merge expected and actual results ([#pr-3960](https://github.com/vector-im/element-android/issues/pr-3960))
```

instead of

```
 - Issue templates: merge expected and actual results ([#3960](https://github.com/vector-im/element-android/issues/3960))
```


Can you rename the file to `3960.misc` please?

Ref: https://github.com/twisted/towncrier